src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -200,482 +198,306 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitDescribeQuery(
-      SparqlAutomaticParser::DescribeQueryContext* ctx) override {
+  antlrcpp::Any visitDescribeQuery(Parser::DescribeQueryContext* ctx) override {
+    // TODO: unsupported
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitAskQuery(
-      SparqlAutomaticParser::AskQueryContext* ctx) override {
+  antlrcpp::Any visitAskQuery(Parser::AskQueryContext* ctx) override {
+    // TODO: unsupported
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitDatasetClause(
-      SparqlAutomaticParser::DatasetClauseContext* ctx) override {
+  antlrcpp::Any visitDatasetClause(Parser::DatasetClauseContext* ctx) override {
+    // TODO: unsupported
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitDefaultGraphClause(
-      SparqlAutomaticParser::DefaultGraphClauseContext* ctx) override {
+      Parser::DefaultGraphClauseContext* ctx) override {
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitNamedGraphClause(
-      SparqlAutomaticParser::NamedGraphClauseContext* ctx) override {
+      Parser::NamedGraphClauseContext* ctx) override {
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitSourceSelector(
-      SparqlAutomaticParser::SourceSelectorContext* ctx) override {
+      Parser::SourceSelectorContext* ctx) override {
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitWhereClause(
-      SparqlAutomaticParser::WhereClauseContext* ctx) override {
+  antlrcpp::Any visitWhereClause(Parser::WhereClauseContext* ctx) override {
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitSolutionModifier(
-      SparqlAutomaticParser::SolutionModifierContext* ctx) override {
+      Parser::SolutionModifierContext* ctx) override {
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitGroupClause(
-      SparqlAutomaticParser::GroupClauseContext* ctx) override {
+  antlrcpp::Any visitGroupClause(Parser::GroupClauseContext* ctx) override {
```

Comment:
I think we can also add a `using Any = antlrcpp::Any` alias (even if we try to throw it out),
then some more of those generic lines get a little shorter, + it costs you about 5 seconds:)